openbas-api/src/main/java/io/openbas/migration/V2_76__Add_reply_to_scenarios.java

```diff
@@ -0,0 +1,26 @@
+package io.openbas.migration;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+import java.sql.Statement;
+
+@Component
```

Comment:
Can you mutualize this migration ? To have one migration for exercise & scenario ?

Reply:
yes!